priv/repo/migrations/20230718175315_add_sni_host.exs

```diff
@@ -0,0 +1,15 @@
+defmodule Supavisor.Repo.Migrations.AddSniHost do
+  use Ecto.Migration
+
+  def up do
+    alter table("tenants", prefix: "_supavisor") do
+      add(:sni_hostname, :string, null: true)
+    end
+  end
+
+  def down do
+    alter table("tenants", prefix: "_supavisor") do
+      remove(:sni_hostname)
+    end
+  end
```

Comment:
I think you can write this migration with only a `change` function:

```suggestion
  def change do
    alter table("tenants", prefix: "_supavisor") do
      add(:sni_hostname, :string, null: true)
    end
  end
```

WDYT?

Reply:
looks good, thanks!